src/webextension/background/datastore.js

```diff
@@ -4,6 +4,11 @@
 
 import * as DataStore from "lockbox-datastore";
 
-const datastore = DataStore.create({});
+let datastore;
 
-export default datastore;
+export default async function runit(fn) {
```

Comment:
Nit: I'd call this function something like `withDatastore`. I'm also not sure about passing a function to it; wouldn't it be better to simply return the datastore object? Then it's easier to do stuff like use `Promise.all()` with this function.

Reply:
I changed it to `openDataStore()` and returning a Promise.